tests/CMakeLists.txt

```diff
@@ -1,9 +1,12 @@
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${GLOW_BINARY_DIR}/tests/)
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/unittests)
```

Comment:
Do we need this? Does `add_subdirectory(unittests)` below not handle this?

Reply:
Yeah, the actually sets the `-I` include path so that it contains the headers in tests/unittests, whereas `add_subdirectory` just tells CMake to traverse that subdirectory.